android/app/src/main/java/updated/mysterium/vpn/model/filter/NodeQuality.kt

```diff
@@ -0,0 +1,16 @@
+package updated.mysterium.vpn.model.filter
+
+import network.mysterium.proposal.QualityLevel
+
+enum class NodeQuality {
+    LOW, MEDIUM, HIGH;
+
+    companion object {
+
+        fun parseFromQualityLevel(qualityLevel: QualityLevel) = when (qualityLevel) {
```

Comment:
Rename this to `from(quality: QualityLevel)`

Reply:
Fix